userspace/async/async_key_value_source.tpp

```diff
@@ -150,27 +150,32 @@ bool async_key_value_source<key_type, value_type>::lookup(
 	}
 
 	typename value_map::const_iterator itr = m_value_map.find(key);
-	bool request_complete = (itr != m_value_map.end()) && itr->second.m_available;
+	bool request_complete;
 
-	if(!request_complete)
+	if (itr == m_value_map.end())
 	{
 		// Haven't made the request yet
-		if (itr == m_value_map.end())
-		{
-			m_value_map[key].m_available = false;
-			m_value_map[key].m_value = value;
-		}
+		m_value_map[key].m_available = false;
+		m_value_map[key].m_value = value;
 
 		// Make request to API and let the async thread know about it
 		if (std::find(m_request_set.begin(),
-		              m_request_set.end(),
-		              key) == m_request_set.end())
+			      m_request_set.end(),
```

Comment:
Uber nit: Did you change spaces to tabs on these two lines?  Seems like you ought not do that :)

Reply:
Bad CLion, no cookie